### ISSUE TITLE
Error when opening a file in vi which contains the substring "vimrc" in its filename

### ISSUE THREAD

**Issue by gmaghera** [skwp/dotfiles#622]
When I open a file which has "vimrc" in its filename, in the Terminal (not in GUI mode), I get an error:

Here is the command using the file name "stringcontaining**vimrc**throwsanerror":
```shell
$ mvim -v stringcontainingvimrcthrowsanerror
```

And the resulting output:
```shell
"stringcontainingvimrcthrowsanerror" [New File]
Error detected while processing /Users/maghga/.yadr/vim/bundle/vim-textobj-function/after/ftplugin/vim/textobj-functio
n.vim:
line   30:
E128: Function name must start with a capital or "s:": g:textobj_function_vim_select(object_type)
line   31:
E133: :return not inside a function
line   78:
E700: Unknown function: g:textobj_function_vim_select
Press ENTER or type command to continue
```

Maybe this is an error with vim or MacVim in general, but I'm not sure.

**Comment by gmaghera** [comment]
I've experimented a little more with this, and the issue is not limited to files with "vimrc" in their filename.  Maybe it's with the vim syntax detection.  I've created a vim script with the mkview command, and opening that file results in the same error.

**Comment by gmaghera** [comment]
Has anyone been able to reproduce this error?  Is it something specific to my environment?  

I've found a similar issue when googling for the error message, but I can't quite tell if it's in the same exact plugin:

https://github.com/thinca/vim-textobj-function-javascript/issues/2

**Comment by gmaghera** [comment]
I started getting similar errors when updating YADR, and stumbled across the fix here:

https://github.com/VundleVim/Vundle.vim/issues/470

```
cd ~/.vim/bundles/vundle
git pull
```

I'm not sure how I got into a state where I had an older version of vundle, since I don't manipulate it outside of YADR.  From the lack of responses to this issue, I'm guessing it was some sort of oddball error specific to my environment.  Anyway, here is the fix, in case someone else finds it useful.
